lib/npg_qc/autoqc/checks/pulldown_metrics.pm

```diff
@@ -142,33 +172,40 @@ override 'execute' => sub {
     return 1;
 };
 
+has 'output_file' => (
+    is => 'ro',
+    isa => 'Str',
+    lazy_build => 1,
+);
+
+sub _build_output_file {
+    my $self = shift;
+    return catfile($self->qc_out->[0], $self->filename_root.'_gatk_collecthsmetrics.txt');
```

Comment:
I suggest a dot between the filename root and gatk in line with other naming conventions, for example, `40216#4_phix.markdups_metrics.txt`. Also, count we have `gatk_collecthsmetrics.txt` as a constant so that it's easily visible on the top of the file?